integration_tests/steps/device.py

```diff
@@ -18,13 +18,13 @@ def step_impl(context):
 def step_impl(context, setup_file):
     sleep(3)
     system(f'docker run -d --name d3a-tests --env REDIS_URL=redis://redis.container:6379/ --net integtestnet '
-           f' d3a-tests -l INFO run -t 1s -s 60m --setup {setup_file} --no-export')
+           f' d3a-tests -l INFO run -t 5s -s 60m --setup {setup_file} --no-export')
```

Comment:
This change influences the results. So one potential reason for the failing tests, also, i would suggest to add `--seed 0`